src/components/CalendarLayout/_components/Calendar/Calendar-styles.less

```diff
@@ -3,31 +3,50 @@
 	display: flex;
 	justify-content: center;
 	align-items: center;
+	color: #4a4a4a;
 }
 
 .events {
 	margin: 0;
 	padding: 0;
 	list-style: none;
+	color: #4a4a4a;
 }
+
 .events .ant-badge-status {
 	width: 100%;
 	overflow: hidden;
 	font-size: 12px;
 	text-overflow: ellipsis;
+	color: #4a4a4a;
 }
 .notes-month {
 	font-size: 28px;
 	text-align: center;
+	color: #4a4a4a;
 }
 .notes-month section {
 	font-size: 28px;
+	color: #4a4a4a;
 }
 
 .ant-picker-panel {
 	.ant-picker-date-panel,
 	.ant-picker-body,
 	.ant-picker-content {
 		height: 100%;
+		color: #4a4a4a;
 	}
 }
+:where(.css-dev-only-do-not-override-u24u6l).ant-list .ant-list-item {
+	display: block;
+	padding: 5px;
+}
+
+:where(.css-dev-only-do-not-override-u24u6l).ant-list-split .ant-list-item:last-child {
```

Comment:
Ne **JAMAIS** utiliser les classes `.css-dev-only-do-not-override`, le nom de la classe est assez explicite.